components/02-elements/form/checkbox/_checkbox.scss

```diff
@@ -9,11 +9,31 @@ $checkbox__icon-fill            : $color-primary !default;
 $checkbox__icon-transition      : $transition-base !default;
 $checkbox__border               : 2px solid $form-elements-border-color !default;
 $checkbox__border--checked      : none !default;
+$checkbox__color--fake          : $gray-dark !default;
 
 .checkbox {
     position: relative;
     margin-bottom: $checkbox__margin;
 
+    &--link {
+        display: block;
+        text-decoration: none;
+        color: $checkbox__color--fake;
+
+        &:hover {
+            text-decoration: underline;
+        }
+        &-checked {
```

Comment:
I think we should use `&--link-checked` and move it to proper level and not creating thinks like modifier inside modifier.

Reply:
Done